Speedykom/Regional-Pandemic-Analytics
# Feat: Add Vault service

Jira Ticket: https://speedykom.atlassian.net/browse/REPAN-433

## Description
This PR implements a new feature that enables storing all secrets (environment variables ) inside HCP Vault, this offers a higher protection for secret keys, passwords etc

## Changes Made
Add HCP Vault service and store env variables under JSON format
Add script for generating .env files 
Add .env files to .gitignore

## Testing Instructions
- after starting the project, run `make start-vault`
- run `./gen-env.sh`
- verify that .env files are succesfully generated

## Screenshots
![image](https://github.com/Speedykom/Regional-Pandemic-Analytics/assets/142224614/ad227e46-7b47-42f0-9bf5-892f5af6a163)
